docs/api.md

```diff
@@ -3386,8 +3377,7 @@ Kills the browser process.
 #### browserApp.wsEndpoint()
 - returns: <?[string]> Browser websocket url.
 
-Browser websocket endpoint which can be used as an argument to [browserType.connect(options)] to establish connection to the browser. Requires browser app to be launched with `browserType.launchBrowserApp({ webSocket: true, ... })`.
-
+Browser websocket endpoint which can be used as an argument to [browserType.connect(options)] to establish connection to the browser.
```

Comment:
Missing the `(#browsertypeconnectoptions)`.

Reply:
Done